lib/tasks/locale.rake

```diff
@@ -320,6 +320,14 @@ namespace :locale do
       end
 
       Rake::Task['gettext:po_to_json'].invoke
+
+      # Work-around for extraneous '\' symbols in json catalogs
+      # This can go away once https://github.com/webhippie/gettext_i18n_rails_js/issues/42 is resolved
+      Dir.glob(File.join(GettextI18nRailsJs::Task.output_path, "*.json")).each do |file|
```

Comment:
Changed this line since it wasn't finding the .json files correctly before

Reply:
@DavidResende0 What is the procedure for testing this?

Reply:
I ran `bundle exec rake locale:po_to_json` which updates the `.json` files and then checked to make sure:
1. The strings in the files have been correctly updated/"fixed"
2. The translations begin to appear in ManageIQ